src/test/java/org/jenkinsci/plugins/workflow/cps/actions/ArgumentsActionImplTest.java

```diff
@@ -621,6 +626,22 @@ public void testReallyUnusualStepInstantiations() throws Exception {
                 equalTo(NotStoredReason.UNSERIALIZABLE));
     }
 
+    @Issue("JENKINS-47101")
+    @Test public void passwordParametersSanitized() throws Exception {
```

Comment:
@car-roll This is the new test I added for https://github.com/jenkinsci/workflow-support-plugin/pull/110.